modules/base/rendering/renderableplane.cpp

```diff
@@ -262,6 +316,33 @@ void RenderablePlane::render(const RenderData& data, RendererTasks&) {
     cameraOrientedRotation[1] = glm::dvec4(newUp, 0.0);
     cameraOrientedRotation[2] = glm::dvec4(normal, 0.0);
 
+    if (_scaleByDistance) {
+        const glm::dvec3 cameraPosition = data.camera.positionVec3();
+        const glm::dvec3 modelPosition = data.modelTransform.translation;
+
+        const float fovDegrees = global::windowDelegate->getHorizFieldOfView();
+        const float fovRadians = glm::radians(fovDegrees);
+        const float halfFovTan = std::tan(fovRadians * 0.5f);
+
+        const float distance = glm::distance(cameraPosition, modelPosition);
+
+        // Projected height based on distance and FOV
+        float projectedHeight = 2.0f * distance * halfFovTan * _scaleRatio.value();
+        projectedHeight = std::clamp(
+            projectedHeight,
+            _scaleByDistanceMinHeight.value(),
+            _scaleByDistanceMaxHeight.value()
+        );
+
+        glm::vec2 currentSize = _size.value();
+
+        if (currentSize.y != 0.f) {
```

Comment:
```suggestion
        if (currentSize.y > 0.f) {
```
? don't think it may be negative?